Gems/TestAssetBuilder/Code/Source/Builder/TestDependencyBuilderComponent.cpp

```diff
@@ -154,21 +156,33 @@ namespace TestAssetBuilder
 
         if (!buffer.empty())
         {
-            bool result = false;
-            AZ::Data::AssetInfo assetInfo;
-            AZStd::string watchFolder;
-            AzToolsFramework::AssetSystemRequestBus::BroadcastResult(result, &AzToolsFramework::AssetSystemRequestBus::Events::GetSourceInfoBySourcePath, buffer.c_str(), assetInfo, watchFolder);
+            AZStd::vector<AZStd::string> tokens;
+            AZ::StringFunc::Tokenize(buffer, tokens, "|");
```

Comment:
Curious: It seems that the buffer may contain multiple paths separated by "|" while previously it only contains one path. Is this for another bug fix? I don't see any other places that may change the buffer value in this PR.

Reply:
I actually updated this as part of these changes for the sake of being able to test these changes manually.  You can see the assets in the screenshot above, but I can't include them since they're essentially broken and would annoy everyone using automated test project with a popup error